package.json

```
@@ -6,9 +6,10 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.4.5",
-    "@catho-private/catho-components": "^5.8.2",
+    "@catho-private/catho-components": "^5.10.0",
     "@catho/quantum": "^9.0.1",
```

```suggestion
    "@catho/quantum": "^9.2.0",
```

should update quantum version.